cmd/dcos/main.go

```diff
@@ -35,6 +50,22 @@ func main() {
 	}
 }
 
+// logLevel returns the log level for the CLI.
```

Comment:
You can add that the verbosity is 0 by default, this is not very explicit in `var verbosity int`.